<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectedProjectWidget</class>
 <widget class="QWidget" name="SelectedProjectWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>58</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="background">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>950</width>
         <height>600</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QGroupBox{
border-radius: 22px;
border:none;
background-color: rgba(255,255,255,180);
}
QGroupBox::hover{
border-radius: 22px;
border:none;
background-color: rgb(255,255,255);
}</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>47</number>
        </property>
        <property name="topMargin">
         <number>47</number>
        </property>
        <property name="rightMargin">
         <number>47</number>
        </property>
        <property name="bottomMargin">
         <number>47</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="icon_bot_button">
            <property name="minimumSize">
             <size>
              <width>54</width>
              <height>54</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>54</width>
              <height>54</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../bot_icons.qrc">
              <normaloff>:/icons/widgets/times_icon/new_boit.png</normaloff>:/icons/widgets/times_icon/new_bot.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>47</width>
              <height>47</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="name_bot_edit">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>58</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>26</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit{
color:black;
border:none;
background:transparent;
}</string>
            </property>
            <property name="text">
             <string>New BOT</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QRadioButton" name="switch_activated_bot">
              <property name="styleSheet">
               <string notr="true">QRadioButton{}

QRadioButton::indicator
{
    width: 50px;
    height:50px;
}

QRadioButton::indicator:checked{
image:url(:/icons/images/start_bot.svg);
}

QRadioButton::indicator:unchecked{
image:url(:/icons/images/stop_bot.svg);
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="marker_state_bot">
              <property name="minimumSize">
               <size>
                <width>119</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QLabel{
border-radius:6px;
border:none;
color:white;
background-color:#FF5F8F;
}</string>
              </property>
              <property name="text">
               <string>Bot is disabled</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="BotView" name="bot_view"/>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="open_in_redactor_button">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>13</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-color: rgb(57,178,146);
border:none;
color:white;
border-radius:8px;
}
</string>
            </property>
            <property name="text">
             <string>Open in redactor -&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>57</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BotView</class>
   <extends>QGraphicsView</extends>
   <header>constructor_app.graphic_scene.bot_view</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../bot_icons.qrc"/>
 </resources>
 <connections/>
</ui>
